### PR TITLE
Update zhCN.lua

### PR DESCRIPTION
**Pull request by nanjuekaien1**

### CODE REVIEW

File: Dragonflight/HallsOfInfusion/Locales/zhCN.lua [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
-	--L.refti_defender = "Refti Defender"
-	--L.dazzling_dragonfly = "Dazzling Dragonfly"
-	--L.curious_swoglet = "Curious Swoglet"
-	--L.flamecaller_aymi = "Flamecaller Aymi"
-	--L.primalist_earthshaker = "Primalist Earthshaker"
-	--L.gusting_protodragon = "Gusting Proto-Dragon"
-	--L.oceanic_protodragon = "Oceanic Proto-Dragon"
-	--L.earthen_protodragon = "Earthen Proto-Dragon"
-	--L.infuser_sariya = "Infuser Sariya"
+        L.primalist_ravager = "拜荒掠夺者"
```
Comment (ntowle) [inline Dragonflight/HallsOfInfusion/Locales/zhCN.lua:4]: Convert spaces to a tab here, please.

File: Dragonflight/AlgetharAcademy/Locales/zhCN.lua [hunk 2]
```diff
@@ -2,22 +2,22 @@ local L = BigWigs:NewBossLocale("Algeth'ar Academy Trash", "zhCN")
 if not L then return end
 if L then
 	L.recruiter_autotalk = "自动对话"
-	--L.recruiter_autotalk_desc = "Instantly pledge to the Dragonflight Recruiters for a buff."
+	L.recruiter_autotalk_desc = "立即找巨龙招募人员获取增益"
 	L.critical_strike = "+5% 爆击"
 	L.haste = "+5% 急速"
-	L.mastery = "+精通"
+	L.mastery = "+97精通"
```
Comment (ntowle) [inline Dragonflight/AlgetharAcademy/Locales/zhCN.lua:8]: We can't include the number in the message, 97, because it scales based on level or difficulty.

**Comment by ntowle** [comment]
Thanks!

**Comment by nanjuekaien1** [comment]
Thanks!

**Pull request merged**